test/integration/load_stats_integration_test.cc

```diff
@@ -703,5 +741,103 @@ TEST_P(LoadStatsIntegrationTest, DropOverloadDropped) {
   cleanupLoadStatsConnection();
 }
 
+// Validate the load reports for successful requests as cluster membership
+// changes.
+TEST_P(LoadStatsIntegrationTest, SuccessWithCustomMetrics) {
```

Comment:
Please also add a test that ensures that the metrics are not populated, if they are not populated (i.e., that the entire `load_metric_stats` field isn't present).

Reply:
done